test/test_autograd.py

```diff
@@ -7198,8 +7198,32 @@ class TestAutogradForwardModeBatchedGrad(TestCase):
     def test_out_of_place_basic(self):
         a = torch.rand(4, 4, dtype=torch.double, requires_grad=True)
         b = torch.rand(4, 4, dtype=torch.double, requires_grad=True)
-        self.assertTrue(gradcheck(torch.sin, a))
-        self.assertTrue(gradcheck(torch.add, (a, b)))
+        self.assertTrue(gradcheck(torch.sin, a, check_forward_ad=True, check_batched_grad=True,
+                                  check_batched_forward_grad=True))
+        self.assertTrue(gradcheck(torch.add, (a, b), check_forward_ad=True, check_batched_grad=True,
+                                  check_batched_forward_grad=True))
```

Comment:
Err did we ever make it so that there are only `check_batched_forward_grad` and `check_batched_reverse_grad` instead of needing to do this strange dance where we have to set both `{check_batched_grad, check_batched_forward_grad}` to True?

Reply:
Not yet! It is currently planned as a follow up because I wanted to think about things that can be done to improve how OpInfo attributes are passed into gradcheck https://github.com/pytorch/pytorch/pull/66294#discussion_r726118804. I should do that soon...